src/datachain/telemetry.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from importlib.metadata import PackageNotFoundError, version
+
+from iterative_telemetry import IterativeTelemetryLogger
+
+from datachain.utils import env2bool
+
+logger = logging.getLogger(__name__)
+
+
+def is_enabled():
+    """
+    Determine if telemetry is enabled based on environment variables and configuration.
+    """
+    # Disable telemetry if running in test mode
+    if env2bool("DATACHAIN_TEST"):
+        return False
+
+    # Check if telemetry is disabled by environment variable
+    disabled = env2bool("DATACHAIN_NO_ANALYTICS")
```

Comment:
In dvc, this envvar disables analytics when it is set, regardless of it's value. 

```suggestion
    disabled = bool(os.getenv("DATACHAIN_NO_ANALYTICS"))
```

The envvar being set is an enough signal from the user, and the boolean values are confusing (is it `1`, `true`, `True`, `0`, 'y', 'yes', etc to disable analytics?).